gramps/plugins/gramplet/filter.py

```diff
@@ -52,7 +59,7 @@ class Filter(Gramplet):
     The base class for all filter gramplets.
     """
 
-    FILTER_CLASS = None
+    FILTER_CLASS: Any = None
```

Comment:
I think this could be
```suggestion
    FILTER_CLASS: type[SidebarFilter] | None = None
```
It may even be a good idea to wrap in `ClassVar`.

Reply:
I think this didn't work; anyway the goal here is not to *add* type hints consistenly yet, but just to make mypy pass to *enable* the addition of type hints from now on, so I'll leave this here for now.

Reply:
@DavidMStraub Understood, but please be aware that changing `Any` to **literally anything else** is a BC break. Based on my experience, `Any` should be used sparingly (even when you think it's what you need, most of the time you actually need `type` or `object` instead).